test/arguments.js

```diff
@@ -0,0 +1,41 @@
+import test from 'ava';
+import shell from 'shelljs';
+import tempfile from 'tempfile';
```

Comment:
Actually you are right, it's not *strictly* needed anymore since all the commits *should* happen with mocked git. However, should the mock not work they wont, ending up committing the original project. This could also be substituted by giving commit an invalid argument, so that it never actually goes through, whether git is mocked or not? The real problem is that, on windows the test files modify the same global path, ending up with flaky tests where you're not sure if the git is mocked or not.. Then this is a real problem without some safeguard.